web/template/payments.gohtml

```diff
@@ -16,8 +16,34 @@
                 <table class="govuk-table my-table table__no-border">
                     <tbody class="govuk-table__body">
                     <tr class="govuk-table__row">
-                        <th scope="row" class="govuk-table__header govuk-!-font-size-24">Total paid:</th>
-                        <td class="govuk-table__cell govuk-!-font-size-24"><strong>£{{ fee .TotalPaid }}</strong></td>
+                        {{ if gt .OutstandingFee 0 }}
+                            <td class="govuk-table__header govuk-!-padding-top-3 govuk-!-padding-bottom-8 govuk-!-font-size-24 app-!-border-bottom-black">Outstanding fee due:</td>
```

Comment:
This still makes sense as a `th` since it scopes the row:
```suggestion
                            <th scope="row" class="govuk-table__header govuk-!-padding-top-3 govuk-!-padding-bottom-8 govuk-!-font-size-24 app-!-border-bottom-black">Outstanding fee due:</th>
```